health-sdk/sdk/src/main/java/net/gini/android/health/sdk/paymentcomponent/PaymentComponent.kt

```diff
@@ -88,7 +88,8 @@ class PaymentComponent(private val context: Context, private val giniHealth: Gin
                     LOG.debug("Loaded payment providers")
                     LOG.debug("Loading installed payment provider apps")
 
-                    val paymentProviderApps = getPaymentProviderAppsSorted(paymentProvidersResource.data)
+                    val supportedPaymentProviderApps = paymentProvidersResource.data.filter { it.gpcSupportedPlatforms?.contains("android") == true || it.openWithSupportedPlatforms?.contains("android") == true }
```

Comment:
[Request] This is not needed now that you are filtering using `PaymentProvider.isEnabled()`.

Reply:
Yes, forgot about leaving this in - removed it.